lib/Dialect/Calyx/CalyxOps.cpp

```diff
@@ -40,6 +40,16 @@ static LogicalResult verifyProgramOp(ProgramOp program) {
 // ComponentOp
 //===----------------------------------------------------------------------===//
 
+/// Gets the WiresOp of a component.
+static WiresOp getWiresOp(ComponentOp componentOp) {
+  WiresOp wiresOp;
+  auto body = componentOp.getBody();
+  auto opIt = llvm::find_if(*body, [](auto &&op) { return isa<WiresOp>(op); });
```

Comment:
I'm not familiar at all with this dialect, but this looks O(n) to me, it would be good to reflect that in the doc comment.

Reply:
The WiresOp is a container for all wire assignments and GroupOps, e.g. 
```
calyx.component @C () -> () {
  %0 = instance0
  %1 = instance1
  ...
  calyx.wires { ... }
  calyx.control { ... }
}
```

To further clear this up, the `WiresOp` should generally be the second-to-last op in a ComponentOp. At this time we aren't explicitly requiring an ordering, so just selecting the `body.end() - 2`th operation is incorrect.